RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -8,6 +8,10 @@
 
 #include "RecoTauTag/RecoTau/interface/DeepTauBase.h"
 
+namespace deep_tau {
+    constexpr int NumberOfOutputs = 4;
```

Comment:
why not `static` as previously?

Reply:
`static` is redundant. Before `static` was needed because it was defined inside `struct dnn_inputs_2017v1`

Reply:
Yes, you are right